recipe/meta.yaml

```diff
@@ -41,14 +41,32 @@ test:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    #
+    # Also skip `test_rolling_var_numerical_issues` on `ppc64le` as it is a known test failure.
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/149
+    {% set extra_args = ["-m not clipboard and not single_cpu", "--skip-slow", "--skip-network", "--skip-db", "-n=2"] %}
+    {% set extra_args = extra_args + ["-k not test_rolling_var_numerical_issues"] %}   # [ppc64le]
```

Comment:
Skipping `test_rolling_var_numerical_issues` on Linux `ppc64le`. Have filed issue ( https://github.com/conda-forge/pandas-feedstock/issues/149 ) to track.

Reply:
Also some of the `ppc64le` jobs are rather long running. Are there more tests we could cut here?

Reply:
Looks like there is [an `arm_slow` marker]( https://github.com/pandas-dev/pandas/blob/v1.5.2/pyproject.toml#L61 ). Going to try skipping that assuming it would be helpful here as well